FlowCrypt/Functionality/Services/Local Private Key Services/CombinedPassPhraseStorage.swift

```diff
@@ -87,19 +87,19 @@ final class CombinedPassPhraseStorage: CombinedPassPhraseStorageType {
 
     init(
         encryptedStorage: PassPhraseStorageType,
-        inMemoryStorage: PassPhraseStorageType = InMemoryPassPhraseStorage()
+        inMemoryStorage: PassPhraseStorageType
```

Comment:
And here in original code

Reply:
We can't keep original code because `InMemoryPassPhraseStorage` now requires `encryptedStorage`

Reply:
@tom